File: App_Santander_Test/app/src/main/java/com/ivan/bankapp/model/Statements.java
package com.ivan.bankapp.model;

import com.google.gson.annotations.SerializedName;

public class Statements {

    @SerializedName("title")
    private String title;

    @SerializedName("desc")
    private String desc;

    @SerializedName("date")
    private String date;

    @SerializedName("value")
    private Double value;

    public Statements(String tittle, String desc, String date, Double value) {
        this.title = tittle;
        this.desc = desc;
        this.date = date;
        this.value = value;
    }

    public String getTittle() {
        return title;
    }

    public void setTittle(String tittle) {
        this.title = tittle;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Double getValue() {
        return value;
    }

    public void setValue(Double value) {
        this.value = value;
    }
}

File: BankApp/app/src/main/java/com/paulokeller/bankapp/data/models/Statements.kt
package com.paulokeller.bankapp.data.models

import com.google.gson.annotations.SerializedName


class Statement(
    @SerializedName("title") val title: String,
    @SerializedName("desc") val desc: String,
    @SerializedName("date") val date: String,
    @SerializedName("value") val value: Double
)

class Statements(@SerializedName("statementList") val results: List<Statement>)